### PR TITLE
Move cities list to the left

### PR DESCRIPTION
**Pull request by EmmanuelDemey**
The cities list is not visible on mobile. We have to scroll a lot in order to be able to select a city. So I propose to move this list on the left side of the page .

The list of baggers is now sorted by the name.

### CODE REVIEW

**Review by dadoonet — APPROVED** [review]
LGTM but I'd really like to get @linsolas thoughts... :D

File: js/vue-app.js [hunk 1]
```diff
@@ -17,6 +17,16 @@ new Vue({
       body: null
     }
   },
+  computed: {
+    displayedBaggers: function(){
+      var filtered = this.filtered(this.baggers);
+      return filtered.sort(function(bagger1, bagger2){
```
Comment (linsolas) [inline js/vue-app.js:23]: why not using `bagger1.name.localeCompare(bagger2.name)`?

**Comment by linsolas** [comment]
Except for the `sort` function, I'm ok with the PR.

**Comment by dadoonet** [comment]
Ping @Gillespie59: wanna apply changes proposed by @linsolas?

**Comment by EmmanuelDemey** [comment]
Yep. Will commit the changes today or tomorrow

**Comment by EmmanuelDemey** [comment]
Done @dadoonet  @linsolas

**Pull request merged**